### PR TITLE
Soccer Magazine

### PR DESCRIPTION
**Pull request by levy002**
## Description
In this PR, we created The Soccer Magazine mobile application.

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, We:

- [x] fetched data from API.
- [x] dispatched the actions to store the selected data in the Redux store.
- [x] used `useSelector()` Redux Hook to consume data from Redux store.
- [x] displayed a list of categories that can be filtered out by player position.
- [x] displayed the player details when the user clicked on the player profile.
- [x] created `home` button to navigate to `homepage`.
- [x] tested the website components by `jest` library.
- [x] styled the website components with Talwind.CSS

### Video
Please watch [this](https://www.loom.com/share/64c6998b0f294533834458d8c42a4b25) video which explains briefly the app functionalities.

### Live Demo
 Please Visit Live Demo [here](https://soccer-magazine-202.netlify.app/)

 ### Happy reviewing 😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
-# fish_species
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Meltrust) [inline README.md:1]: - [ ] On your readme, kindly please add the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  The link featured in that section should point to a file at the root of your project (the license file).   Just copy it from the mentioned template:

image

That way, we will meet this requirement, and your project will be well documented 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @levy002,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Your project is beautiful and works as intended✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (levy002) [reply README.md:1]: Thank @Meltrust  for your suggestions, now it's addressed. :handshake:

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @levy002 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by levy002** [comment]
> ## **STATUS: APPROVED  **
> ### Hi @levy002 ,
> **Great work on making the changes requested by a previous reviewer 👏🏻**
> 
> Your project is complete! . There is nothing else to say other than... it's time to merge it champagneship :shipit: . Congratulations! tada100star2
> 
> 5SW
> 
> **Cheers and Happy coding! clap clap clap**
> 
> Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you @ShoiraTa :handshake:

**Pull request merged**